manifests/ad-block-updater-FD176DD1-F9A0-4469-B43E-B1764893DD5C-manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "Brave Ad Block Updater extension",
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAoqpe6QWKketr66AW+hKSxr5qds9gxmU72N20bG/nG8wfbPUdTwEqIG3m5e37Iq1FI0gcQir5UqwZZUgkum5dWJwgCB1SOaVvlZrWZbTbATKFcePswHqAIXMfS+wzMA/Ifoky26dE4k/rs6J/zQEbeXXon/ikjGJ7GxYeHYMBz9DAPQhcUoBlh1C0O0vhvXU+kG5DO4wdIt9W/cXJtv+8OTZ6HiTJw1j0jAliFZI/jhkYB6MW57OBpBYlWJQhMbLbK5opXq6d4ELbjC1amqI1lT3j5bl0g1OpMqL4Jtz6578G79gMJfxE3hA5tL0rGU3vAmwck/jXh7uOOzqetwdBcwIDAQAB",
+  "manifest_version": 2,
+  "name": "Brave Ad Block Updater (BGR: Bulgarian Adblock list)",
+  "version": "0.0.0"
+}
```

Comment:
Can we use the ad-block node native lib to generate these automatically from the ad-block list, or are you still having trouble with getting it built as a dep?

Reply:
Good idea, I'll look into it. Haven't tried your tip yet about `GYP_MSVS_VERSION`, I'll give that a shot!

Reply:
Fixed in newest commit.